<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(78, 154, 6);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionRevert"/>
    <addaction name="separator"/>
    <addaction name="actionPrint_Preview"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionTime"/>
    <addaction name="separator"/>
    <addaction name="actionFont"/>
    <addaction name="actionColor"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUpdate"/>
    <addaction name="actionAbout_NotePad"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/new.png</normaloff>:/new/icon/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New(&amp;Ctrl+N)</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/open.png</normaloff>:/new/icon/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open(&amp;Ctrl+O)</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/save.png</normaloff>:/new/icon/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save(&amp;Ctrl+S)</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/save.png</normaloff>:/new/icon/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save As((&amp;Ctrl++Shift+N))</string>
   </property>
  </action>
  <action name="actionRevert">
   <property name="text">
    <string>Revert</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="text">
    <string>Print Preview(&amp;Ctrl+Shift+N)</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print...(&amp;Ctrl+P)</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit(&amp;Ctrl+Q)</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close(&amp;Ctrl+W)</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo(&amp;Ctrl+Z)</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo(&amp;Ctrl+Y)</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/copy.png</normaloff>:/new/icon/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy(&amp;Ctrl+C)</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/cut.png</normaloff>:/new/icon/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut(&amp;Ctrl+X)</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/paste.png</normaloff>:/new/icon/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste(&amp;Ctrl+V)</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All(&amp;Ctrl+A)</string>
   </property>
  </action>
  <action name="actionTime">
   <property name="text">
    <string>Time</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionAbout_NotePad">
   <property name="text">
    <string>About NotePad</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
